app/templates/components/popular-tags.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="col-md-3">
+  <div class="sidebar">
```

Comment:
The thing is that this component is only for Popular Tags as the templating for article tags is a bit different. It could be possible to reuse but I am not sure this is necessary